### DISCUSSION TITLE
branding page for Revolt website

### DISCUSSION THREAD

**Discussion by Andre601** [stoatchat/discussions#175] [General]
The Revolt website should receive a branding page that explains how the logo can be used, how copyright and stuff works, what laws and rules apply, etc.

The reason for this is a (somewhat heated) discussion I had with mods on the Revolt Server, talking about if I'm allowed to use the "Revolt" watermark (The text) for an image. While I got the permission from Insert, do I feel like I didn't need it to begin with, since the watermark is basic geometric forms and/or text/font which makes it not copyrightable and instead public domain (At least this is my basic understanding).

Having a branding page could really help answering questions like:

- Can I use the logo/watermark?
- What is the copyright/license of them?
- What are the brand colours?
- Where/What can I NOT use the logo in?
- Who can I contact for advice and permission?

I'm not sure how the page can or should look like, especially since I have no knowledge of how the website is even set up to PR my idea of a branding page...

Maybe with some help can I come up with a rough idea of the page.

**Comment by MagicLike** [comment]
Also these assets should be available for download.
